### PR TITLE
Hover text

### LINKED ISSUE #33: Add hover text to navigation bar

### PR DESCRIPTION
**Pull request by Holo-23**

### CODE REVIEW

File: packages/frontend/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -21,32 +21,39 @@ export enum Page {
...
   },
   {
     page: Page.SOUNDBOARD,
     iconPath: 'navigation/SoundIcon.svg',
+    text: 'Soundboard',
   },
   {
     page: Page.CREDITS,
     iconPath: 'navigation/InfoIcon.svg',
+    text: 'Credits',
```
Comment (RedMaps) [inline packages/frontend/components/Navigation/Navigation.tsx]: You could just use the page as the hover text, as i've assigned the enum strings. (fanart doesnt have a plural btw)

File: packages/frontend/components/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -69,6 +76,7 @@ export default function Navigation({
           {navItems.map((item) => (
             <Link key={item.iconPath} href={item.page.toString()}>
               <div className={styles.icon}>
+                <text className={styles.icon_text}>{item.text}</text>
```
Comment (RedMaps) [inline packages/frontend/components/Navigation/Navigation.tsx]: You could do something like 
```
item.page.charAt(0).toUpperCase() + item.page.slice(1);
```

**Review by RedMaps — CHANGES_REQUESTED** [review]
I didn't expect this to get implemented like this, but somehow i really like it

File: packages/frontend/components/Navigation/Navigation.tsx (hunk 1 above)
Comment (Holo-23) [reply packages/frontend/components/Navigation/Navigation.tsx]: Oh yeah lol I can do that I guess

**Comment by H-aisha23** [comment]
Is this fine?

**Comment by Holo-23** [comment]
https://github.com/Kaichou-Project/kaichou-graduation/blob/7aa54ea8e7d2bb78b3de60eed2bc0457c26e246b/packages/frontend/components/Navigation/Navigation.tsx#L71-L75
Is this fine?

**Review by RedMaps — APPROVED** [review]
Looks good! Thanks for the contribution!

**Pull request merged**